### PR TITLE
Award-Array

### PR DESCRIPTION
**Pull request by Amandev02**
## Issue that this pull request solves

Closes: #131

## Proposed changes

Brief description of what is fixed or changed
implemented array for Awards
## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by Amandev02** [comment]
Screenshot 2022-04-03 155600

**Comment by praveenscience** [comment]
This is #131 and not related to #131. Which issue does it close?

File: src/components/App.js [hunk 1]
```diff
@@ -21,7 +21,11 @@ const App = () => {
       },
       Experience: [],
       Education: [],
-      Awards: [],
+      Awards: {
+        Award_Name: "your award name",
+        Recieved : "Recieved year",
+        Description: "short description of your award",
+      },
```
Comment (praveenscience) [inline src/components/App.js:28]: What if a person has got multiple awards?

Comment (Amandev02) [reply src/components/App.js:28]: Yes sir , i thought about that later ,give some time i will update it

Comment (praveenscience) [reply src/components/App.js:28]: @Amandev02 Will wait for it. But can you first discuss?

Comment (Amandev02) [reply src/components/App.js:28]: I was thoughting to get the no of awards and implement a loop n times

Comment (praveenscience) [reply src/components/App.js:28]: No no, that's not the right way. Check how the others are implemented. Have you worked with React?

Comment (Amandev02) [reply src/components/App.js:28]: Or can u suggest something

Comment (praveenscience) [reply src/components/App.js:28]: We just need to know what all the different properties, a single object can contain. That's all.

Comment (Amandev02) [reply src/components/App.js:28]: i m new bie in react let me try

**Pull request closed without merging**